layouts/partials/contact.html

```diff
@@ -44,7 +44,8 @@ <h2>{{ .title }}</h2>
       <div class="col-12">
           <div class="block">
             <h2>Book a meeting</h2>
-            <div class="calendarhero-inline-widget" data-url="https://meeting.calendarhero.com/meeting/new/6140a9c858a75c0020d98e26/meeting" style="min-width:320px; height:630px;"></div>
+            <p>You can quickly book a meeting with Bas or call yourself using the button below:</p>
+            <a href="https://app.reclaim.ai/m/bas" target="_blank"><button>Schedule Meeting</button></a>
```

Comment:
@LFSb I can probably reskin the button by going with a `<a class="btn btn-info mt-lg-2" ...>` like in the `get-started.html` page..

Reply:
The current button does look a little windows-95 😅

Reply:
Decided to not wait for input and just fix it. Looks better IMHO 😉

Reply:
<img width="735" alt="image" src="https://user-images.githubusercontent.com/5371367/207891993-7b3fc3b4-2d03-4d65-aadd-74eaa9d892d7.png">
A _lot_ better, yes 😄